test/e2e/aio/command_test.go

```diff
@@ -232,6 +308,9 @@ func TestCommandServices(t *testing.T) {
 			t.Run("then a ready Keystone StatefulSet should be created", func(t *testing.T) {
 				assert.NoError(t, w.ForReadyStatefulSet("commandtest-keystone-keystone-statefulset", 1))
 			})
+			t.Run("then a ready Config StatefulSet should be created", func(t *testing.T) {
```

Comment:
Could you also add check in e2e test TestCluster that command can proxy requests to config and analytics services?

Here we test if requests are sent via command proxy to keystone https://github.com/Juniper/contrail-operator/blob/2b8f86038699426b893f5c3e0f02dfdd6e0c5af9/test/e2e/aio/command_test.go#L339

Reply:
Done